charts/incubator/hyperswitch-app/templates/producer/secret.yaml

```diff
@@ -26,4 +26,7 @@ data:
   ROUTER__SECRETS__ADMIN_API_KEY: {{ .Values.application.server.secrets.admin_api_key | b64enc }}
   ROUTER__SECRETS__JWT_SECRET: {{ .Values.application.server.secrets.jwt_secret | b64enc }}
   ROUTER__SECRETS__MIGRATION_ENCRYPTION_TIMESTAMP: {{ "1685510751" | b64enc }}
+  ROUTER__CRM__TOKEN: {{ .Values.application.server.secrets.crm_token | b64enc }}
+  ROUTER__LOCKER__REDIS_TEMP_LOCKER_ENCRYPTION_KEY: {{ .Values.application.server.secrets.redis_temp_locker_encryption_key | b64enc }}
```

Comment:
Can we please add generic envs for router, consumer, and producer in `misc/secrets`?

Reply:
yeah, we ll do the refactoring in separate PR

Reply:
yes, let us do all the refactoring in a separate PR including configs.